tests/devapps/XamarinDev/XamarinDev.Android/XamarinDev.Android.csproj

```diff
@@ -61,7 +61,12 @@
     <PackageReference Include="Xamarin.Android.Support.v7.MediaRouter" Version="28.0.0.3" />
     <PackageReference Include="Xamarin.Android.Support.Core.Utils" Version="28.0.0.3" />
     <PackageReference Include="Xamarin.Android.Support.CustomTabs" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Essentials" Version="1.0.1" />
+    <PackageReference Include="Xamarin.Essentials" Version="1.6.0" />
+	<PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.1.0" />
+    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.0.0" />
+    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0" />
+    <PackageReference Include="Xamarin.AndroidX.MediaRouter" Version="1.1.0" />
```

Comment:
do we need all these additional packages?

Reply:
These are to update the test app to androidX. Figured it is worth updating to the latest to test out the flows.
However, not needed

Reply:
I'm asking more because if customers were to update MSAL, would they also have to install these packages. if they are not part of the broker updates, then i think it's okay. 

---
In reply to: [570607603](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2380#discussion_r570607603) [](ancestors = 570607603)

Reply:
@neha-bhargava can we double check what gets added when updating the Xamarin samples? thanks.

Reply:
@jennyf19 I updated the Xamarin samples sometime back for testing and these updates looks fine for androidX.